test/Confluent.Kafka.UnitTests/Admin/CreateTopicsError.cs

```diff
@@ -0,0 +1,70 @@
+// Copyright 2022 Confluent Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Refer to LICENSE for more information.
+
+using Xunit;
+using System;
+using System.Collections.Generic;
+using Confluent.Kafka.Admin;
+
+namespace Confluent.Kafka.UnitTests
+{
+    public class CreateTopicsErrorTests
```

Comment:
I think i would just make this `AdminClient` - we don't have any other admin client unit tests, and are unlikely to make many (because there's not much you can do).

great idea to make this a unit test, not integration test.